change/react-native-windows-2020-05-26-19-03-03-fixcommon.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "Remove references to ReactWindowsCore projects",
```

Comment:
>Remove references [](start = 14, length = 17)

Nit: Looks like it's not just the references that get removed :)

Reply:
Ended up scaling this down. The sources will stay in that folder for the time being, but the projects are indeed removed.